File: Sheet 7 (Arrays 2)/L.cpp
#include <iostream>
using namespace std;

int main() {
    int n; cin >> n; int time = 0;
    int arr[n]; for (int i = 0; i < n; i++) cin >> arr[i];
    
    if (n % 2 != 0) {
        cout << "still running" << endl;
        return 0;
    }
    
    for (int i = 0; i < n; i += 2) {
        time += arr[i + 1] - arr[i];
    }

    cout << time << endl;
    return 0;
}

File: Sheet 7 (Arrays 2)/L.py
n = int(input())
ans = 0
ok = 0
pre = 0

for i in range(n):
    x = int(input())
    if (not ok):
        ok = 1
        pre = x
    else:
        ok = 0
        ans += x - pre

if (ok):
    print('still running')
else:
    print(ans)
